common/libs/web-workers/workers/scrypt-worker.worker.ts

```diff
@@ -0,0 +1,18 @@
+import { fromV3, IFullWallet } from 'ethereumjs-wallet';
+
+declare var postMessage: any;
```

Comment:
From https://github.com/webpack-contrib/worker-loader#integrating-with-typescript:

```
const ctx: Worker = self as any;
ctx.postMessage({ foo: "foo" });
```
for typing